dll/win32/shell32/shresdef.h

```diff
@@ -270,9 +270,11 @@
 #define IDS_FONTS_DESCRIPTION      22920
 #define IDS_ADMINISTRATIVETOOLS_DESCRIPTION 22921
 #define IDS_FOLDER_OPTIONS_DESCRIPTION 22924
+#define IDS_TASKBAR_OPTIONS_INFOTIP 30348
 #define IDS_ADMINISTRATIVETOOLS    22982
 #define IDS_FOLDER_OPTIONS         22985
 #define IDS_FONTS                  22981
+#define IDS_TASKBAR_OPTIONS        32517
```

Comment:
Do these ID values attempt to match Windows' ?

Reply:
Yes